libs/core/src/lib/select/option/option.component.ts

```diff
@@ -61,81 +110,149 @@ export class OptionComponent implements OnInit, OnDestroy {
     @Input()
     disabled: boolean = false;
 
+    @Output()
```

Comment:
This needs a comment for documentation.

Reply:
@salarenko This was the specific thing that I documented in the documentation as difference between previous version current version..